sql/hive/pom.xml

```diff
@@ -135,6 +135,11 @@
       <artifactId>jackson-mapper-asl</artifactId>
     </dependency>
     <!-- transitive dependencies of hive-exec-core doesn't declare -->
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>${javaxservlet.version}</version>
```

Comment:
Do we need this line, `<version>`? I believe we can remove this line.

Reply:
Yes we need the version, or else I will need to declare the same under  `dependencyManagement`  for `spark-parent`.
Should I declare once the on spark-parent so all modules can reuse the declaration?

Reply:
Yes, please put it into `spark-parent` to make it sure we use a single version.

Reply:
Moved both javax and Jakarta version to parent, waiting for the CI.

Reply:
After the change, both servlet-api got removed from `deps/spark-deps-hadoop-3-hive-2.3`, checking.

Reply:
Marking this as resolved.